source/developers-guide/attribute-system/index.md

```diff
@@ -903,3 +903,99 @@ s_articles_attributes_my_column_label = "Deutsches label"
+
+and use these information to create recommendations in his account section.  
+To demonstrate how this works we have the following example plugin.
+### Bootstrap
+`SwagAttributeSlider/SwagAttributeSlider.php`:
+```
+<?php
+
+namespace SwagAttributeSlider;
+
+use Shopware\Bundle\AttributeBundle\Service\CrudService;
+use Shopware\Components\Plugin;
+use Shopware\Components\Plugin\Context\InstallContext;
+
+class SwagAttributeSlider extends Plugin
+{
+    /**
+     * @param InstallContext $context
+     */
+    public function install(InstallContext $context)
+    {
+        /** @var CrudService $crud */
+        $crud = $this->container->get('shopware_attribute.crud_service');
+
+        $crud->update('s_user_attributes', 'recommendedVariants', 'multi_selection', [
+            'displayInBackend' => true,
+            'label' => 'Recommended variants',
+            'entity' => 'Shopware\Models\Article\Detail',
+        ]);
+        $crud->update('s_user_attributes', 'recommendedStream', 'single_selection', [
+            'displayInBackend' => true,
+            'label' => 'Recommended stream',
+            'entity' => 'Shopware\Models\ProductStream\ProductStream',
+        ]);
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            'Enlight_Controller_Action_PostDispatch_Frontend_Account' => 'onPostDispatchAccount',
+        ];
+    }
+
+    /**
+     * @param \Enlight_Event_EventArgs $args
+     */
+    public function onPostDispatchAccount(\Enlight_Event_EventArgs $args)
+    {
+        /** @var $controller \Shopware_Controllers_Frontend_Detail */
+        $controller = $args->get('subject');
+        $controller->View()->addTemplateDir($this->getPath() . '/Resources/views');
+    }
+}
+```
+We create two new attributes in the `s_user_attributes` table:
+* `single_selection` column mapped to the `Shopware\Models\ProductStream\ProductStream` entity for the product stream slider
+* `multi_selection` column mapped to the `Shopware\Models\Article\Detail` entity for product variant slider
+
+We also subscribe to the `Enlight_Controller_Action_PostDispatch_Frontend_Account` event to register our template.
+### Template extension
+`SwagAttributeSlider/Resources/views/frontend/account/index.tpl`
+```
+{extends file="parent:frontend/account/index.tpl"}
+{block name="frontend_account_index_welcome"}
+	{$smarty.block.parent}
+
+	{$data = $sUserData.additional.user}
+
+	<h2>Recommended variants for you</h2>
+
+	{action module=widgets controller=listing action=productSlider numbers=$data.recommendedvariants}
+
+	<h2>Recommended stream</h2>
+
+	{action module=widgets controller=listing action=streamSlider streamId=$data.recommendedstream}
+
+{/block}
+```
+In the template we call the widgets actions and handle over the attribute data.  
+Thats it. We can now choose individual products or streams which will be shown to the customer.
```

Comment:
Thats it. Now choose individual products or streams which will be shown to the customer.